lib/core/test/unit/Cardano/Wallet/Primitive/AddressDerivationSpec.hs

```diff
@@ -366,7 +381,7 @@ genRootKeys :: Gen (Key 'RootK XPrv)
 genRootKeys = do
     (s, g, e) <- (,,)
         <$> genPassphrase @"seed" (16, 32)
-        <*> genPassphrase @"generation" (0, 16)
+        <*> genPassphrase @"generation" (10, 16)
```

Comment:
I don't think this is right? Where does this lower bound comes from for the generation passphrase? It certainly can be empty, and I don't think there's a minimum length at all?

Reply:
because of that - https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs#L246
I reverted this